### PR TITLE
stat/distuv: provide full test coverage for statdist.go and increase numerical accuracy of Hellinger distance

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Please take a look.

### CODE REVIEW

File: stat/distuv/statdist_test.go [hunk 1]
```diff
@@ -144,6 +144,21 @@ func TestKullbackLeiblerBeta(t *testing.T) {
 			t.Errorf("Kullback-Leibler mismatch, case %d: got %v, want %v", cas, got, want)
 		}
 	}
+	ok := Beta{0.5, 0.5, nil}
```
Comment (kortschak) [inline stat/distuv/statdist_test.go:147]: s/ok/good/g

`ok` is conventionally a boolean.

File: stat/distuv/statdist_test.go [hunk 2]
```diff
@@ -193,3 +208,59 @@ func klSample(samples int, l RandLogProber, r LogProber) float64 {
...
+		{
+			a: Beta{Alpha: 0.5, Beta: 0.4, Src: rnd},
+			b: Beta{Alpha: 0.7, Beta: 0.2, Src: rnd},
+		},
+		{
+			a: Beta{Alpha: 3, Beta: 5, Src: rnd},
+			b: Beta{Alpha: 5, Beta: 3, Src: rnd},
+		},
+	} {
+		want := math.Sqrt(1 - math.Exp(-Bhattacharyya{}.DistBeta(test.a, test.b)))
```
Comment (kortschak) [inline stat/distuv/statdist_test.go:232]: nit: Please have got, *then* want for these lines, and in the condition.

File: stat/distuv/statdist_test.go [hunk 3]
```diff
@@ -193,3 +208,59 @@ func klSample(samples int, l RandLogProber, r LogProber) float64 {
...
+		{
+			a: Normal{Mu: 0, Sigma: 2, Src: rnd},
+			b: Normal{Mu: 2, Sigma: 2, Src: rnd},
+		},
+		{
+			a: Normal{Mu: 0, Sigma: 5, Src: rnd},
+			b: Normal{Mu: 2, Sigma: 0.1, Src: rnd},
+		},
+	} {
+		want := math.Sqrt(1 - math.Exp(-Bhattacharyya{}.DistNormal(test.a, test.b)))
```
Comment (kortschak) [inline stat/distuv/statdist_test.go:260]: Same here.

**Comment by romanwerpachowski** [comment]
PTAL.

**Pull request merged**